Landing CV/index.html

```diff
@@ -125,9 +125,11 @@ <h2 class="skills_name">Skills</h2>
             </div>
         </div>
                 <!----------Skills_hide---------->
+        <input type="checkbox" class="checker" id="checker"/>
+        <div class="label_box">
+            <label for="checker" class="label_button"></label>
+        </div>
```

Comment:
the button should be under all skills
![image](https://user-images.githubusercontent.com/20189313/130373552-13c84546-44f6-4843-95d6-096bad1b6e3b.png)